lib/protobuf/generators/field_generator.rb

```diff
@@ -20,7 +20,7 @@ class FieldGenerator < Base
       attr_reader :field_options
 
       def applicable_options
-        @applicable_options ||= field_options.map { |k, v| ":#{k} => #{v}" }
```

Comment:
Dang it @zachmargolis beat me while I had typed out my answer but failed to press send for 20 minutes :P